Source/CarthageKit/FoundationExtensions.swift

```diff
@@ -0,0 +1,48 @@
+
+import Foundation
+
+extension Collection where Element: Hashable {
+    
+    func unique() -> [Element] {
+        var set = Set<Element>(minimumCapacity: count)
+
+        return filter {
+            return set.insert($0).inserted
+        }
+    }
+}
+
+extension FileManager {
+    
+    public func allDirectories(at directoryURL: URL, ignoringExtensions: Set<String> = []) -> [URL] {
+        func isDirectory(at url: URL) -> Bool {
+            let values = try? url.resourceValues(forKeys: [.isDirectoryKey])
+            return values?.isDirectory == true
+        }
+        
+        let options: FileManager.DirectoryEnumerationOptions = [.skipsHiddenFiles]
+        let keys: [URLResourceKey] = [.isDirectoryKey]
+        guard
+            directoryURL.isFileURL,
+            isDirectory(at: directoryURL),
+            let enumerator = self.enumerator(at: directoryURL, includingPropertiesForKeys: keys, options: options)
+        else
+        {
```

Comment:
Not sure about the bracket style here. (I know, great comment ;D )

Reply:
Fixed to match existing style